tests/queries.ts

```diff
@@ -184,3 +184,13 @@ testClient(async function transactionWithConcurrentQueries() {
     assertEquals(r.rows[0][0], i);
   });
 });
+
+testClient(async function handleNameTooLongError() {
+  const result = await CLIENT.queryObject(`
+    SELECT id AS "very_very_very_very_very_very_very_very_very_very_very_long_name" FROM ids ORDER BY id
```

Comment:
Please don't use the ids table for the test (I'm trying to get rid of the fixtures used for the tests and use temporally generated ones instead). Do something like `SELECT 1 AS very_very_very_very_very_very_very_very_very_very_very_long_name`